src/defaultPhrases.js

```diff
@@ -0,0 +1,176 @@
+const closeDatePicker = 'Close';
+const focusStartDate = 'Focus on start date';
```

Comment:
this phrase feels a little off to me - without running this, I have a hard time understanding what it is this is describing. If I were to hear this phrase being said, I wouldn't understand why I would need to focus on the start date. Seems like this should be something more like "Select start date", but I'm not sure if that's exactly what this is accomplishing.